docs/robots.txt.template

```diff
@@ -0,0 +1,10 @@
+User-Agent: *
+Sitemap: http://docs.soliditylang.org/sitemap.xml
+
+Allow: /en/latest/
+Allow: /en/v0.7.6/
+Allow: /en/v{{ LATEST_VERSION }}/
+
+# Prevent documentation for the development branches and older Solidity
+# versions from showing up in search results.
+Disallow: /en/*
```

Comment:
I haven't verified, but assume Allow+Disallow this way works.

Reply:
@axic yes, because the checks are tested from top to bottom for each matching group.

Reply:
But does it rely just on the order or does it choose the most specific match? Google's docs say it uses the first most specific match for user agent so I guess the same should be true for `Allow`/`Disallow`? They don't say it explicitly though.

Maybe we'll soon find out anyway ;)